src/features/search/pages/modals/DatesHoursModal/DatesHoursModal.tsx

```diff
@@ -333,28 +320,8 @@ export const DatesHoursModal: FunctionComponent<DatesHoursModalProps> = ({
                   <Controller
                     control={control}
                     name="selectedHours"
-                    render={({ field: { value } }) => (
-                      <View>
-                        <Spacer.Column numberOfSpaces={4} />
-                        <LabelHoursContainer nativeID={hoursLabelId}>
-                          <Typo.Body>{`Sortir entre`}</Typo.Body>
-                          <Typo.ButtonText>{`${value?.[0]}\u00a0h et ${value?.[1]}\u00a0h`}</Typo.ButtonText>
-                        </LabelHoursContainer>
-                        <Spacer.Column numberOfSpaces={2} />
-                        <Slider
-                          showValues={false}
-                          values={value}
-                          max={MAX_HOUR}
-                          formatValues={formatHour}
-                          onValuesChangeFinish={onSelectedHoursValuesChange}
-                          minLabel="Horaire minimum de sortie&nbsp;:"
-                          maxLabel="Horaire maximum de sortie&nbsp;:"
-                          shouldShowMinMaxValues
-                          minMaxValuesComplement="h"
-                          sliderLength={sliderLength}
-                          accessibilityLabelledBy={hoursLabelId}
-                        />
-                      </View>
+                    render={({ field: { value, onChange } }) => (
+                      <HoursSlider defaultValue={value as [Hour, Hour]} onChange={onChange} />
                     )}
```

Comment:
si on peut se passer de l'état interne de HoursSlider, on pourrait améliorer comme ça

```suggestion
                    render={HoursSlider}
```